src/components/Discussion/Internal/Comment/IconLink.js

```diff
@@ -62,18 +56,34 @@ export const IconLink = ({
     marginLeft: small ? 0 : 20,
     ...style
   }
+  const linkStyleRule = useMemo(
+    () =>
+      css({
+        color: colorScheme.getCSSColor('primary'),
+        fill: colorScheme.getCSSColor('primary'),
+        '@media (hover)': {
+          ':hover': {
+            color: colorScheme.getCSSColor('primaryHover'),
+            fill: colorScheme.getCSSColor('primaryHover')
+          }
+        }
+      }),
+    [colorScheme]
+  )
 
   return (
-    <a href={href} onClick={onClick} {...styles.link} style={patchedStyle}>
+    <a
+      href={href}
+      onClick={onClick}
+      {...styles.link}
+      {...linkStyleRule}
+      style={patchedStyle}
+    >
       <span {...styles.icon}>
-        <Icon size={dimension} {...colorScheme.set('fill', 'primary')} />
```

Comment:
why remove fill here?

Reply:
The color is now set through the `linkStyleRule`. It includes a hover state and fixes the hover color, which was different from the other links.

Reply:
👍🏼